ibm_cloud_sdk_core/iam_token_manager.py

```diff
@@ -61,6 +62,7 @@ class IAMTokenManager(JWTTokenManager):
     """
     DEFAULT_IAM_URL = 'https://iam.cloud.ibm.com/identity/token'
```

Comment:
I think this should not include the path part, right?
```suggestion
    DEFAULT_IAM_URL = 'https://iam.cloud.ibm.com'
```